docs/.vuepress/styles/index.styl

```diff
@@ -1,3 +1,15 @@
+// temporary search bar fix
+.search__container
+  visibility visible!important
+
+.layout__main__content__aside__container
+  margin-top 60px
+
+#aside-scroll .container .search__container
+  display none
+
+
```

Comment:
Lgtm, works locally...I'd remove the extra whitespace and move this down below the original block.

Reply:
I added the whitespace to mark the end of the temporary fix block and didn't want to put this in the middle of the file (before the `@media (min-width: 700px)`) - but so you would move it to the end of the file? Happy to adjust that.

Reply:
this sounds like a nit that might have justification anyway. Since it's the only comment from an otherwise LGTM I'm gonna merge it 👍